README.md

```diff
@@ -6,6 +6,7 @@
     1. [Toolchain](building/toolchain.md)
     2. [Building script](building/script.md)
     3. [Reference project repository](building/project.md)
+    4. [Setting up Windows](building/windows.md)
```

Comment:
I'm not sure if I'd add that here. Wouldn't a reference in building/README.md suffice? @damianloew

Reply:
I think we can leave it there.

Reply:
Yeah, I think it is not necessary to place it in the main table of contents.